crowdsec-docs/unversioned/getting_started/healthcheck.mdx

```diff
@@ -0,0 +1,173 @@
+
+*Trigger CrowdSec's test scenarios*  
+
+:::info
+Check that your Security Engine properly reads and parses the logs of the services you protect.  
+The HTTP collection and the Linux collection contain test scenarios allowing you to prove your Security Engine works as intended, without having to do penetration tests, risking getting you banned from your own server.
+:::
+
+### Test HTTP detection
+
+For this test, you'll add the following path to a URL of a website you're protecting: `/CRODWSEC/TEST`  
+_*// Any issue possible with .htaccess that would rewrite before this path reaches the log? – **Yes, this is a valid concern. Add a note: "Important: Ensure your web server configuration (e.g., .htaccess) does not rewrite or block this specific URL path `/CRODWSEC/TEST`."***_  
+Looking at your alerts for the specific scenario **crowdsec/test-http**, you should see a line related to your test.  
+Note that this scenario has a delay of 1 minute before it can re-trigger _(blackhole parameter of the scenario)_.  You can use the "-a" flag to list all alerts.  
+
+<CodeBlock className="language-bash">sudo cscli alerts list -s crowdsec/test-http -a</CodeBlock>
+
+### Test SSH detection
+
+Try to authenticate to your server via SSH using the following user: `ABCDEFGH123456`.  
+Looking at your alerts for the specific scenario **crowdsec/test-ssh**, you should see a line related to your test.  
+Note that this scenario has a delay of 5 minute before it can re-trigger _(blackhole parameter of the scenario)_.  You can use the "-a" flag to list all alerts.  
+
+<CodeBlock className="language-bash">sudo cscli alerts list -s crowdsec/test-ssh -a</CodeBlock>
+
+### Test AppSec detection
+
+If you are using an AppSec-capable bouncer and have configured CrowdSec WAF, you can test it by [insert specific instructions based on typical AppSec setup; see notes below].  
+_[**Important Notes for AppSec Test:** The specific test and instructions here are heavily dependent on the WAF/bouncer being used (nginx-bouncer, cloudflare-bouncer, etc.) and the specific AppSec rules enabled.  You need a generic but useful example. Suggestions:*  
+- *If possible, install a VERY basic "test XSS" AppSec rule as part of a starter collection.* The test could be adding `<script>alert('XSS')</script>` as a query parameter value in your web application.
+- *If targeting a specific bouncer/WAF, provide instructions for a known trigger for a common rule.* For example, if using the Cloudflare bouncer, provide a query that triggers a known SQL injection or XSS rule.]
+
+* * *
+
+## Ready for next phase ?
+
+Were all the tests related to your setup successful ?
+If so, you can proceed to the next phase of the health check: [**Connectivity checks**](#crowdsec-connectivity-checks).  
+
+If not, check the troubleshooting section below
+
+## 🚨 Detection Troubleshooting
+*No alerts triggered? Let's investigate: Here are some tests to identify where the issue might be.*  
+
+<details>
```

Comment:
I would suggest first doing `cscli metrics show acquisition` and add a screenshot showcasing expected behavior and faulty behavior. The `cscli metrics show parsers` would come later as a further investigation if needed.

Reply:
we'll see how it look in it's final form, but for now it allows to one stone two birds and not have one more level of digging.